scraper/scraper/settings.py

```diff
@@ -13,9 +13,8 @@
     'scraper.pipelines.ResolutionPipeline': 300,
 }
 
-CONCURRENT_REQUESTS = 15
-CONCURRENT_REQUESTS_PER_DOMAIN = 30
-DOWNLOAD_DELAY = 0.50 # ms
+CONCURRENT_REQUESTS = 30
```

Comment:
from the 10-minute i ran i did, it looks like they do. maybe they previously crashed because we made bad requests.